tests/testthat/test-deparse.R

```diff
@@ -250,11 +250,16 @@ test_that("literal lists are deparsed", {
   expect_identical(sexp_deparse(list(TRUE, b = 2L, 3, d = "4", as.raw(5))), "<list: TRUE, b = 2L, 3, d = \"4\", <raw: 05>>")
 })
 
-test_that("long vectors are truncated", {
+test_that("long vectors are truncated by default", {
   expect_identical(sexp_deparse(1:10), "<int: 1L, 2L, 3L, 4L, 5L, ...>")
   expect_identical(sexp_deparse(as.list(1:10)), "<list: 1L, 2L, 3L, 4L, 5L, ...>")
 })
 
+test_that("long vectors are not truncated when max_len = NULL", {
```

Comment:
Can you add a test for the 0 boundary case please?

Reply:
Done (and it didn't work correctly in that case, so I fixed it)